js/script.js

```diff
@@ -0,0 +1,53 @@
+$(document).ready(function() {
+	$('#reset').hide();
+	$('#searchBar').on('submit', function(e) {
+		e.preventDefault();
+		$('.pageStart').hide();
+		$('#reset').show();
+		var $searchRequest = $('#searchBar input').val();
+		
+		var recycle = 0;
+		
+		$.get('https://www.reddit.com/search.json?q=' + $searchRequest +'+nsfw:no').done(function(data) {
+
+			var results = data.data.children
+
+			// Getting objects from Reddit
+			var getImages = function(url) {
```

Comment:
This works for the first few posts because the URLs are images, but it's possible that this URL can be a link to another site. Try looking at the `thumbnail` or `preview` properties inside each post. Either one should return images.